src/src/Controller/FotogalerieController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+use App\Repository\ReceptyRepository;
+use App\Entity\Recepty;
+
+class FotogalerieController extends AbstractController
```

Comment:
Ideally only use English when writing code - it's the best practice (everyone speaks English these days) and some code-generators need to have properties named in English to generate code properly.